cropduster/models.py

```diff
@@ -427,15 +433,18 @@ def save_size(self, size, thumb=None, image=None, tmp=False, standalone=False,
         if not image and not self.image:
             raise Exception("Cannot save sizes without an image")
 
-        image = image or PIL.Image.open(safe_str_path(self.image.path))
+        if not image:
+            with default_storage.open(self.image.path) as f:
```

Comment:
I think this needs to be `self.image.name` (I suspect that `self.image.path` will work for the `FileSystemStorage` but not the s3 boto storage).

Reply:
Nah, works as is- `Image` here is an `ImageFieldFile` where `path` is [defined](https://github.com/django/django/blob/master/django/db/models/fields/files.py#L55) as `self.storage.path(self.name)`- so it's correctly relative for S3 and absolute for `FileSystemStorage`.

Reply:
(for the record- I was wrong here 😄 - I was testing these changes in an application wherein I created a storage class that overrode the value of `path` to be `name`)